app/src/main/java/com/devshish/internship/presentation/ui/home/charts/artists/ArtistItemAdapter.kt

```diff
@@ -0,0 +1,72 @@
+package com.devshish.internship.presentation.ui.home.charts.artists
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.devshish.internship.R
+import com.devshish.internship.databinding.ItemArtistBinding
+import com.devshish.internship.databinding.ItemListHeaderBinding
+import com.devshish.internship.domain.model.Artist
+import com.devshish.internship.presentation.model.ChartItem
+import com.devshish.internship.presentation.model.ChartItemCallback
+import com.devshish.internship.presentation.ui.home.charts.TitleViewHolder
+
+class ArtistItemAdapter(
+    private val context: Context
+) : ListAdapter<ChartItem, RecyclerView.ViewHolder>(ChartItemCallback()) {
+
+    companion object {
+        private const val TYPE_HEADER = 0
```

Comment:
So here you will simplify your adapter to only work with items and not with additional title